templates/admin/base_site.html

```diff
@@ -1,10 +1,12 @@
 {% extends "admin/base.html" %}
 {% load i18n %}
 
-{% block title %}{{ title }} | {% trans 'Pathagar admin' %}{% endblock %}
+{% block title %}{{ title }} | {{ request.site.name }} {% trans 'admin' %
+    }{% endblock %}
 
 {% block branding %}
-    <h1 id="site-name">{% trans 'Pathagar administration' %}</h1>
+    <h1 id="site-name">{{ request.site.name }}
+        {% trans 'administration' %}</h1>
 {% endblock %}
```

Comment:
I'm wondering if we should just get rid of this file entirely? Doesn't really seem to provide much, and even if it does, we *probably* can now use `urls.py` [for that purpose](http://stackoverflow.com/a/26011790).